app.json

```diff
@@ -33,6 +33,24 @@
     },
     "SMTP_USERNAME":{
       "required":true
+    },
+    "SENDGRID_USERNAME":{
+      "required":true
+    },
+    "SENDGRID_PASSWORD":{
+      "required":true
+    },
+    "SENDGRID_API_KEY":{
+      "required":true
+    },
+    "MAILER_DOMAIN":{
+      "required":true
+    },
+    "MAILER_ADDRESS":{
+      "required":true
+    },
+    "MAILER_SENDER":{
```

Comment:
Can these names be a bit more clear?  Ideally, we shouldn't have to refer to the code to figure out what these are.

Suggestion `DEFAULT_SEND_FROM_ADDRESS`